app/services/fees/calculator_test.go

```diff
@@ -0,0 +1,108 @@
+package fees
```

Comment:
- I think that it will be beneficial to check/verify that the estimated fee is relying on both service fee and tx fee. We can check with variables such as `service-fee` = 0 and `transferFee` >= `transferAmount` or the other way around `service-fee=10%` and `trasnferFee` = 0.
- I think that we should add a test case in which we have a lot of validators, f.e the `transferFee` will be enough if there is 1 validator, but since there are 10, the fee is not enough

Reply:
Currently, this is done because fee tests depend on the configuration.
I would suggest a manual mock of the configuration - declaring a var config with the needed addresses and fee params.